test/correctness/simd_op_check_x86.cpp

```diff
@@ -567,10 +567,11 @@ class SimdOpCheckX86 : public SimdOpCheckTest {
             check("vpminsq", 8, min(i64_1, i64_2));
         }
         if (use_avx512 && target.has_feature(Target::AVX512_SapphireRapids)) {
-            check("vcvtne2ps2bf16*zmm", 32, cast(BFloat(16), f32_1));
-            check("vcvtneps2bf16*ymm", 16, cast(BFloat(16), f32_1));
-            check("vcvtneps2bf16*xmm", 8, cast(BFloat(16), f32_1));
-            check("vcvtneps2bf16*xmm", 4, cast(BFloat(16), f32_1));
+            // TODO: fails LLVM verification in LLVM16
```

Comment:
Should this be an open issue?

Reply:
...yep, that was it, fix pushed, PTAL

Reply:
...Whoops, nope, I was mistaken. There were bugs in can_run_the_code, but this was unrelated. Opened issue https://github.com/halide/Halide/issues/7219 (feel free to claim it).

Reply:
It may take me a few weeks to get to. I'll try to take a look over the holidays!